### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by litchipi**
Mispelled `workflows`, and forgot 2-3 things

Shame *ding ding*
Shame *ding ding*

Normally the CI should show up in this PR before it gets merged, it's my bad I didn't check it in the previous one (#21)
So I'll only merge this one once it shows up

### CODE REVIEW

**Comment by litchipi** [comment]
@AurelienFT There is something going on with the tests, I had to retry 3 times on the CI to get it green (as it worked perfectly on my system)
Maybe allocating a listening port, or anything OS-level can get difficult on the CI for whetever reason.
I guess for now it can be fine to merge this, but that definitely can be annoying if we have to retry X times to get it green.

File: src/lib.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //! This crate abstracts the network layer of a P2P network and provides a simple interface to connect to other peers.
 //! Simple example with two peers on the same code to demonstrate:
-//! ```rust
+//! ```ignore
```
Comment (AurelienFT) [inline src/lib.rs:3]: it's not possible to maintain this up to date ? So that the `lib.rs` the entrypoint of docs always have a working example

Comment (litchipi) [reply src/lib.rs]: Fixed this, I thought it was not intended to really work my bad
(Never heard of doctests before, it's nice ! :smiley: )

**Comment by litchipi** [comment]
@AurelienFT I had to restart the tests 4 times ... It's annoying I don't really like if we have to do this every single time.
Should I disable the annoying test and create an issue about it ? So we can merge this

**Comment by AurelienFT** [comment]
> @AurelienFT I had to restart the tests 4 times ... It's annoying I don't really like if we have to do this every single time. Should I disable the annoying test and create an issue about it ? So we can merge this

Yes

File: tests/simple.rs [hunk 2]
```diff
@@ -30,9 +30,9 @@ fn simple() {
     };
     let mut manager = PeerNetManager::new(config);
     manager
-        .start_listener(TransportType::Tcp, "127.0.0.1:8080".parse().unwrap())
+        .start_listener(TransportType::Tcp, "127.0.0.1:64850".parse().unwrap())
         .unwrap();
-    //manager.start_listener(TransportType::Quic, "127.0.0.1:8081".parse().unwrap()).unwrap();
+    //manager.start_listener(TransportType::Quic, "127.0.0.1:64850".parse().unwrap()).unwrap();
     let clients = create_clients(11);
```
Comment (AurelienFT) [inline tests/simple.rs:36]: You need to pass the port to the client helper creation function if you change the listener port

Comment (litchipi) [reply tests/simple.rs:36]: I added it to the `create_clients` function also

**Comment by AurelienFT** [comment]
It seems that it's maybe the sleeping time that is not long enough. We should not use sleep in the future for tests it was just to prototype. Try to increase a bit the sleeps.

**Review by AurelienFT — APPROVED** [review]
Let's merge it like that. Will help the project to stay stable. Thank you

**Pull request merged**